server/models/user.js

```diff
@@ -1,13 +1,16 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 const userSchema = new Schema({
-
-	id: mongoose.Schema.Types.ObjectId,
-	name: { type: String, required: true } ,
-	email: { type: String, required: true, unique: true } ,
-	password: { type: String, required: true }
-
+  id: mongoose.Schema.Types.ObjectId,
+  name: { type: String, required: true },
+  email: { type: String, required: true, unique: true },
+  password: { type: String, required: true },
+  isChef: { type: Boolean, required: true },
+  location: { type: String, required: true },
+  aboutMe: String,
+  favRecipes: [{ type: Schema.Types.ObjectId, ref: "Recipe" }],
```

Comment:
Nice. This could be useful in the future. From the mocks we also need Favourite Cuisines